pkg/tenant/apikeys.go

```diff
@@ -33,6 +32,7 @@ func (s *Server) ProjectAPIKeyList(c *gin.Context) {
 	}
 
 	// orgID is required to check ownership of the project
+	// TODO: Ensure that the project is in the organization
```

Comment:
Do we have a security problem with this until we fix the project check?

Reply:
Yes we'd have to retrieve the project to do the check, should I add that back in until we have an alternative way to perform the check?

Reply:
I added the checks back in since List/Create API keys is a crucial part of the beta release.